src/modules/assertions/require_cc0.ts

```diff
@@ -0,0 +1,30 @@
+import { context, getOctokit } from "@actions/github";
+import { setFailed } from "@actions/core";
+import {
+  GITHUB_TOKEN,
+  GithubSelf,
+  PR,
+  Review
+} from "src/domain";
+import _ from "lodash";
+
+// BOT will fail if no CC0 detected in PR body
+// Just after: Verify correct environment and request context in main.ts, if no CC0 is detected, 
+// BOT will fail and comment: "error: No CC0 waiver detected".
+
```

Comment:
These comments just seem to describe what the code does, and they are already out of date.  In general, we should strive to write self-documenting code rather than using comments to document what the code does.
```suggestion
```